### PR TITLE
build(extension): add damonsk.vscode-wardley-map

### PR DESCRIPTION
**Pull request by johannrichard**
Add [Wardley Maps extension](https://github.com/42Crunch/vscode-openapi.git) (under MIT License).

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -222,6 +222,11 @@
       "id": "ctf0.macros",
       "repository": "https://github.com/ctf0/macros"
     },
+    {
+      "id": "damonsk.vscode-wardley-maps",
+      "repository": "https://github.com/damonsk/vscode-wardley-maps.git",
+      "version": "1.0.7"
```
Comment (jankeromnes) [inline extensions.json:228]: No need to pin a specific version if we just publish from the default branch (i.e. not from a specific release branch or tag).

```suggestion
      "repository": "https://github.com/damonsk/vscode-wardley-maps"
```

**Review by jankeromnes — APPROVED** [review]
Looks good to me, many thanks for your contribution!

One small adjustment needed, but I can fix that directly on GitHub before merging. 👍

**Comment by johannrichard** [comment]
Thanks! And apologies for the version pinning. I completely missed that in the existing entries. 🤗

**Comment by jankeromnes** [comment]
No worries! `node add-extension <url>` does add a version pin by default, so it's not obvious that you have to manually add a corresponding `"checkout"` or remove the `"version"`.

**Comment by jankeromnes** [comment]
@johannrichard Oops, auto-publishing failed while compiling the extension:

```
> vscode-wardley-maps@1.0.7 compile:ext /tmp/repository
> tsc -b

Error: server/src/server.ts(16,8): error TS2307: Cannot find module 'vscode-languageserver' or its corresponding type declarations.
Error: server/src/server.ts(20,8): error TS2307: Cannot find module 'vscode-languageserver-textdocument' or its corresponding type declarations.
Error: server/src/server.ts(75,52): error TS7006: Parameter '_event' implicitly has an 'any' type.
Error: server/src/server.ts(95,37): error TS7006: Parameter 'change' implicitly has an 'any' type.
Error: server/src/server.ts(119,34): error TS2345: Argument of type 'Thenable<ExampleSettings> | undefined' is not assignable to parameter of type 'Thenable<ExampleSettings>'.
  Type 'undefined' is not assignable to type 'Thenable<ExampleSettings>'.
Error: server/src/server.ts(121,2): error TS2322: Type 'Thenable<ExampleSettings> | undefined' is not assignable to type 'Thenable<ExampleSettings>'.
  Type 'undefined' is not assignable to type 'Thenable<ExampleSettings>'.
Error: server/src/server.ts(125,22): error TS7006: Parameter 'e' implicitly has an 'any' type.
Error: server/src/server.ts(131,30): error TS7006: Parameter 'change' implicitly has an 'any' type.
Error: server/src/server.ts(182,36): error TS7006: Parameter '_change' implicitly has an 'any' type.
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! vscode-wardley-maps@1.0.7 compile:ext: `tsc -b`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the vscode-wardley-maps@1.0.7 compile:ext script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-11-20T03_18_18_817Z-debug.log
ERROR: "compile:ext" exited with 2.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vscode-wardley-maps@1.0.7 vscode:prepublish: `yarn run compile`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the vscode-wardley-maps@1.0.7 vscode:prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2020-11-20T03_18_18_911Z-debug.log
[FAIL] Could not process extension: {
  "id": "damonsk.vscode-wardley-maps",
  "repository": "https://github.com/damonsk/vscode-wardley-maps"
}
npm failed with exit code 1
```

**Comment by johannrichard** [comment]
Ups. I'll have a look at it (and will talk to @damonsk to see whether he's interested to publish on Open VSX as well; and directly, /cc damonsk/vscode-wardley-maps#3)

**Pull request merged**